### PR TITLE
Stop influencing non-doc lines in the docs transformer

### PR DESCRIPTION
**Pull request by webknjaz**
As a side effect, the buggy version was also replacing things in imports that were marked as "keep in core" or "has Base class import"...

### CODE REVIEW

**Comment by webknjaz** [comment]
UPD
===

Nevermind, false alarm. Those cases are processed correctly.

File: migrate.py [hunk 1]
```diff
@@ -217,32 +246,36 @@ def visit_Assign(self, node):
...
 
         # TODO what if it's in a different namespace (different spec)? do we care?
-        new_fragment = '%s.%s.%s' % (args.namespace, fragment_collection, fragment)
-        # TODO make sure to replace only in DOCUMENTATION
-        plugin_data = plugin_data.replace(fragment, new_fragment)
+        new_fragment = f'{namespace}.{fragment_collection}.{fragment}'
+
+        # TODO figure out whether this can be improved
+        doc_val.value = doc_str_tmpl.format(
+            str_val=doc_txt.replace(fragment, new_fragment),
```
Comment (bcoca) [inline migrate.py:245]: shoudl we limit replacements to 1 instance to avoid the 'multipe things rewriten'?

Comment (webknjaz) [reply migrate.py:245]: What do you mean?

Comment (webknjaz) [reply migrate.py:245]: I guess you mean
```suggestion
            str_val=doc_txt.replace(fragment, new_fragment, count=1),
```
But I think that there's a few things to replace there. Anyway, it worked like this before so I'd just keep it until we hit some corner case.

Comment (bcoca) [reply migrate.py:245]: the only issue i see here is if the fragment name is reused in the documentation: `default`, which creates a problem either way, we are 'replacing too much' or 'not enough', this way it should be obvious when it is the latter, the former is probably harder to detect so lets err on 'bad fragment' at this point to discover if we actually have this issue.

Comment (bcoca) [reply migrate.py:245]: really the replace should only happen inside the `extends` pragma in `DOCUMENTATION` but I'm not sure how to code that at this point.

Comment (webknjaz) [reply migrate.py:245]: Should we ignore this for now?

**Pull request merged**